fabric-api-lookup-api-v1/src/testmod/java/net/fabricmc/fabric/test/lookup/InspectorBlock.java

```diff
@@ -46,4 +52,15 @@ public ActionResult onUse(BlockState state, World world, BlockPos pos, PlayerEnt
 
 		return ActionResult.PASS;
 	}
+
+	@Override
+	public void onSteppedOn(World world, BlockPos pos, BlockState state, Entity entity) {
+		Inspectable inspectable = FabricEntityApiLookupTest.INSPECTABLE.find(entity, null);
+
+		if (inspectable != null) {
+			for (ServerPlayerEntity player : world.getServer().getPlayerManager().getPlayerList()) {
```

Comment:
`world.getServer()` returns `null` for client-side world. Probably missing a `!world.isClient` check in the if?

Reply:
Oops, I was using block entity tick before I found about `onSteppedOn` :sweat_smile: